update NEWS.md (#2763)
JuliaData/DataFrames.jl: NEWS.md

# DataFrames.jl changes on main since last release notes

## Performance improvements

* `SubDataFrame` creation is now more efficient if row selector is
  `AbstractVector{Bool}` and selected rows form a continuous block
  ([#2727](https://github.com/JuliaData/DataFrames.jl/pull/2727))

> 

Author: ```suggestion

## Other changes

* `text/plain` rendering of columns containing complex numbers is now improved
  ([#2756](https://github.com/JuliaData/DataFrames.jl/pull/2756))
```

# DataFrames.jl changes on main since last release notes

## Performance improvements

* `SubDataFrame` creation is now more efficient if row selector is

>   `AbstractVector{Bool}` and selected rows form a continuous block

Reviewer: Do we always cast `AbstractVector{Bool}` to `BitVector` in this scenario? I'm asking as `_findall` has been optimized for `B::BitVector` only and calls `Base.findall(B)` otherwise.

Author: ah - right (that is why I wanted you to cross-check :))

Author: actually - maybe it is worth to investigate the same for `AbstractVector{Bool}` case. Would you have time to do it (otherwise I can check).

  ([#2727](https://github.com/JuliaData/DataFrames.jl/pull/2727))

## Other changes

* `text/plain` rendering of columns containing complex numbers is now improved

>   ([#2756](https://github.com/JuliaData/DataFrames.jl/pull/2756))

Author: ```suggestion
  ([#2756](https://github.com/JuliaData/DataFrames.jl/pull/2756))

```

# DataFrames.jl changes on main since last release notes

## Performance improvements

* `SubDataFrame` creation is now more efficient if row selector is

>   `AbstractVector{Bool}` and selected rows form a continuous block

Author: ```suggestion
  `BitVector` and selected rows form a continuous block
```
